content/pages/index.html

```diff
@@ -155,62 +155,51 @@ <h3 id="heading">Test Run for Firefox RHEL6.5 (Desktop)</h3>
                 nextBtn = document.getElementById('next'),
                 prevBtn = document.getElementById('prev'),
                 shown = document.getElementsByClassName('shown')[0],
-                hidden =  document.getElementsByClassName('hidden-right')[0],
+                hidden =  document.getElementsByClassName('hidden')[0],
                 index = document.getElementById('imgIndex');
 
-                index.textContent = (imgIndex + 1) + '/' + imgsLength;
-
-                shown.style.backgroundImage = 'url(' + imgs[0].img + ')';
-                heading.textContent = imgs[0].title;
-                subtitle.textContent = imgs[0].subtitle;
-
-                nextBtn.addEventListener('mouseover', function(){
-                    hidden.className = 'img hidden-right';
-                });
+            index.textContent = (imgIndex + 1) + '/' + imgsLength;
 
-                prevBtn.addEventListener('mouseover', function(){
-                    hidden.className = 'img hidden-left';
-                });
+            shown.style.backgroundImage = 'url(' + imgs[0].img + ')';
+            heading.textContent = imgs[0].title;
+            subtitle.textContent = imgs[0].subtitle;
 
-                nextBtn.addEventListener('click', function(){
-                    imgIndex++;
+            function changeImg(direction) {
+                imgIndex += direction;
 
-                    if(imgIndex > imgsLength - 1){
-                        imgIndex = 0;
-                    }
+                if (imgIndex < 0) {
+                    imgIndex = imgsLength - 1;
+                }
 
-                    index.textContent = (imgIndex + 1) + '/' + imgsLength;
+                if (imgIndex >= imgsLength) {
+                    imgIndex = 0;
+                }
 
-                    shown.className = 'img hidden-right';
-                    hidden.className = 'img shown';
-
-                    hidden.style.backgroundImage = 'url(' + imgs[imgIndex].img + ')';
-                    heading.textContent = imgs[imgIndex].title;
-                    subtitle.textContent = imgs[imgIndex].subtitle;
+                index.textContent = (imgIndex + 1) + '/' + imgsLength;
 
-                    shown = document.getElementsByClassName('shown')[0];
-                    hidden =  document.getElementsByClassName('hidden-right')[0];
-                });
+                shown.classList.replace("shown", "hidden");
```

Comment:
Hmm, I initially though about using `.replace()` but it may not be supported in many browsers. That's why I prefer to have `.remove()` and `.add()` instead although it is ugly as hell.

Reply:
FYI checkout the Browser compatibility table at the Mozilla dev site. Pay attention to the Mobile browsers as well. Looks like replace isn't supported at all on mobile.